scripts/test-runner.js

```diff
@@ -286,6 +286,7 @@ function runTest(name, code, options: PrepackOptions, args) {
   options.invariantLevel = code.includes("// omit invariants") || args.verbose ? 0 : 99;
   if (code.includes("// emit concrete model")) options.emitConcreteModel = true;
   if (code.includes("// exceeds stack limit")) options.maxStackDepth = 10;
+  if (code.includes("// heapGraphFilePath")) options.heapGraphFormat = "VISJS";
```

Comment:
It is not a file path, you are using VISJS format here. I would call "// heap graph" instead of `heapGraphFilePath `

Reply:
Will do. I put it like that originally to avoid confusion with the actual flag (the flag itself is `--heapGraphFilePath` so I kept it the same)

Reply:
Yeah, `heapGraphFilePath` is a command line level option not actually the engine/serializer level option, which is `heapGraphFormat`. But either way, we are just trying to enable heap graph and do not care about the output in this test.